bucket/mpg123.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "1.29.3",
+    "description": "Fast console MPEG Audio Player and decoder library",
+    "homepage": "https://www.mpg123.org",
+    "license": "LGPL-2.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://www.mpg123.org/download/win64/1.29.3/mpg123-1.29.3-static-x86-64.zip",
+            "hash": "a0c082fda8fe4dcdcacbe1a8a58ca367962570e42c07c3532ccb7ac9e4ec87e9",
+            "extract_dir": "mpg123-1.29.3-static-x86-64"
+        },
+        "32bit": {
+            "url": "https://www.mpg123.org/download/win32/1.29.3/mpg123-1.29.3-static-x86.zip",
+            "hash": "cb7ef30ef35697e1d0a34d0430b838f2961e8fb673d7d3d91bf968c8d00cf7a6",
+            "extract_dir": "mpg123-1.29.3-static-x86"
+        }
+    },
+    "bin": [
+        "mpg123.exe",
+        "mpg123-id3dump.exe",
+        "mpg123-strip.exe",
+        "out123.exe"
```

Comment:
What are these 3 files for? Aren't they used internally by mpg123.exe?

Reply:
`mpg123-id3dump.exe`: Tool to dump ID3 meta data from MPEG audio files using libmpg123
`mpg123-strip.exe`: Extract only MPEG frames from a stream using libmpg123
`out123`: is a bit a mistery to me. it's a standalone executable of liboutput123, probably can be used when mpg123 is used as a transcoder. But I think we can cut it off. What do you think?

Reply:
I'm still trying to understand their usage as a standalone tool. Can you provide some usage examples/docs for them?

Reply:
Maybe this could help:

```
> mpg123-id3dump -h
Tool to dump ID3 meta data from MPEG audio files using libmpg123
        version 1.29.3; written and copyright by Thomas Orgis and the mpg123 project

usage: D:\scoop\apps\mpg123\current\mpg123-id3dump.exe [option(s)] file(s)

options:
 -h     --help              give usage help
 -n     --no-scan           do not scan entire file (just beginning)
 -p     --store-pics        write APIC frames (album art pictures) to files
                            file names using whole input file name as prefix
 -v     --verbose           verbose messages from parser

Note that text output will always be in UTF-8, regardless of locale.
```

```
> mpg123-strip -h
Extract only MPEG frames from a stream using libmpg123 (stdin to stdout)
        version 1.29.3; written and copyright by Thomas Orgis and the mpg123 project

usage: D:\scoop\apps\mpg123\current\mpg123-strip.exe [option(s)] < input > output

options:
 -h     --help              give usage help
 -i <n> --icy-interval <n>  stream has ICY metadata present with this interval
 -n     --no-info           also strip info frame at beginning
 -v[*]  --verbose           increase verbosity level
```

```
> out123 -?
Simple audio output with raw PCM input
        version 1.29.3; derived from mpg123 by Michael Hipp and others
        free software (LGPL) without any warranty but with best wishes

usage: out123.exe [option(s)] [file(s) | URL(s) | -]
supported options [defaults in brackets]:
   -v    increase verbosity level       -q    quiet (only print errors)
   -t    testmode (no output)           -s    write to stdout
   -w f  write output as WAV file
   -b n  output buffer: n Kbytes [0]
   -r n  set samplerate [44100]
   -o m  select output module           -a d  set audio device
   -m    single-channel (mono) instead of stereo
   -?    this help                      --version  print name + version
See the manpage out123(1) or call out123.exe with --longhelp for more parameters and information.
```

Reply:
Maybe this could help:

```
> mpg123-strip -h
Extract only MPEG frames from a stream using libmpg123 (stdin to stdout)
        version 1.29.3; written and copyright by Thomas Orgis and the mpg123 project

usage: D:\scoop\apps\mpg123\current\mpg123-strip.exe [option(s)] < input > output

options:
 -h     --help              give usage help
 -i <n> --icy-interval <n>  stream has ICY metadata present with this interval
 -n     --no-info           also strip info frame at beginning
 -v[*]  --verbose           increase verbosity level
```

```
> mpg123-id3dump -h
Tool to dump ID3 meta data from MPEG audio files using libmpg123
        version 1.29.3; written and copyright by Thomas Orgis and the mpg123 project

usage: D:\scoop\apps\mpg123\current\mpg123-id3dump.exe [option(s)] file(s)

options:
 -h     --help              give usage help
 -n     --no-scan           do not scan entire file (just beginning)
 -p     --store-pics        write APIC frames (album art pictures) to files
                            file names using whole input file name as prefix
 -v     --verbose           verbose messages from parser

Note that text output will always be in UTF-8, regardless of locale.
```

```
> out123 -?
Simple audio output with raw PCM input
        version 1.29.3; derived from mpg123 by Michael Hipp and others
        free software (LGPL) without any warranty but with best wishes

usage: out123.exe [option(s)] [file(s) | URL(s) | -]
supported options [defaults in brackets]:
   -v    increase verbosity level       -q    quiet (only print errors)
   -t    testmode (no output)           -s    write to stdout
   -w f  write output as WAV file
   -b n  output buffer: n Kbytes [0]
   -r n  set samplerate [44100]
   -o m  select output module           -a d  set audio device
   -m    single-channel (mono) instead of stereo
   -?    this help                      --version  print name + version
See the manpage out123(1) or call out123.exe with --longhelp for more parameters and information.
```

Reply:
Yes, that was what I was looking for. Thank you!